deploy/charts/trust-manager/templates/deployment.yaml

```diff
@@ -120,7 +121,7 @@ spec:
       {{- end }}
       volumes:
       - name: packages
-        emptyDir: {}
+        {{- toYaml .Values.packagesVolume | nindent 8 }}
```

Comment:
Could you give an example of how and why you would change this value?

Reply:
Hey, just seen this PR - I think we'll go in a different direction with this by setting a size limit unilaterally in #308  - appreciate your involvement. If you wanted to reword this to only focus on revisionHistoryLimit (along with maybe some justification for why we should change that) we'd be happy to take a look!